microshift_updating/microshift-update-rpms-manually.adoc

```diff
@@ -6,11 +6,11 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-Updating {product-title} for non-OSTree systems such as {op-system-base-full} requires downloading then updating the RPMs. For patch releases, such as 4.14.1 to 4.14.2, download and update the RPMs. For minor-version release updates, add the step of enabling the update repository using your subscription manager.
+Updating {product-title} for non-OSTree systems such as {op-system-base-full} requires downloading then updating the RPMs. For patch releases, such as 4.15.1 to 4.15.2, download and update the RPMs. For minor-version release updates, add the step of enabling the update repository using your subscription manager.
 
 [IMPORTANT]
 ====
-{microshift-short} updates are supported from one minor version to the next in sequence. For example, you must update 4.14 to 4.15.
+{microshift-short} updates are supported from one minor version to the next in sequence. For example, you must update 4.14 to 4.15 and so on.
 ====
 
 For either upgrading with patch updates or a minor-version update, you can back up application data as needed and move the data copy to a secure location.
```

Comment:
@ShaunaDiaz The Applying patch updates using RPMs paragraph is awkward. It mentions an example
`For example, use the following procedure to upgrade from 4.14.0 to 4.14.1.`
Maybe replace the 0 with X and 1 with X+1 since the procedure will not actually make that defined update.

Reply:
That change is out of scope of this PR. We're just updating the RHEL version.